pkg/kubelet/rkt/rkt.go

```diff
@@ -62,9 +62,9 @@ const (
 	RktType                      = "rkt"
 	DefaultRktAPIServiceEndpoint = "localhost:15441"
 
-	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "1.2.1"
-	recommendedRktBinVersion = "1.2.1"
+	minimumAppcVersion       = "0.8.1"
```

Comment:
@euank I think so. (Actually the rkt api version is not required as well, as it's always binded to rkt release version) cc @jonboulle 
Let's address this in a follow up PR.